src/vega.schema.ts

```diff
@@ -503,187 +525,187 @@ export type VgBinding = VgCheckboxBinding | VgRadioBinding |
  */
 export interface VgAxisBase {
   /**
-   * A boolean flag indicating if the domain (the axis baseline) should be included as part of the axis.
-   *
-   * __Default value:__ `true`
-   */
-  domain?: boolean;
-
-  /**
-   * A boolean flag indicating if grid lines should be included as part of the axis
-   *
-   * __Default value:__ `true` for [continuous scales](https://vega.github.io/vega-lite/docs/scale.html#continuous) that are not binned; otherwise, `false`.
+   * An interpolation fraction indicating where, for `band` scales, axis ticks should be positioned. A value of `0` places ticks at the left edge of their bands. A value of `0.5` places ticks in the middle of their bands.
    */
-  grid?: boolean;
+  bandPosition?: number;
 
   /**
-   * A boolean flag indicating if labels should be included as part of the axis.
+   * A boolean flag indicating if the domain (the axis baseline) should be included as part of the axis.
    *
-   * __Default value:__  `true`.
+   * __Default value:__ `true`
    */
-  labels?: boolean;
+  domain?: boolean;
 
   /**
-   * Indicates if labels should be hidden if they exceed the axis range. If `false `(the default) no bounds overlap analysis is performed. If `true`, labels will be hidden if they exceed the axis range by more than 1 pixel. If this property is a number, it specifies the pixel tolerance: the maximum amount by which a label bounding box may exceed the axis range.
+   * Stroke width of axis domain line
    *
-   * __Default value:__ `false`.
+   * __Default value:__  (none, using Vega default).
```

Comment:
I think saying "(none, using Vega default)" is almost equivalent to not saying anything as users don't really the information they should know. 

You can look at https://github.com/vega/vega-parser/blob/master/src/config.js to see some defaults.  (`domainWidth` is `1`, for example.)

Same for all other properties. :)